tests/test_exclusion.txt

```diff
@@ -62,4 +62,4 @@
 03-use-cases/03-community-fun/christmas-nyan
 03-use-cases/03-community-fun/monte-carlo-pi
 03-use-cases/03-community-fun/pokemon
-03-use-cases/03-community-fun/wine-analytics
+03-use-cases/03-community-fun/wine-analytics
```

Comment:
Not sure why this is registering as a diff, but this should be the same between `main` and this branch/PR.